admin/src/main/scala/io/buoyant/admin/names/BoundNamesHandler.scala

```diff
@@ -0,0 +1,18 @@
+package io.buoyant.admin.names
+
+import com.twitter.finagle.Service
+import com.twitter.finagle.http.{MediaType, Request, Response}
+import com.twitter.util.{Activity, Future}
+import io.buoyant.namer.EnumeratingNamer
+
+class BoundNamesHandler(namers: Seq[EnumeratingNamer]) extends Service[Request, Response] {
+  override def apply(req: Request): Future[Response] = {
+    Activity.collect(namers.map(_.getAllNames)).map { names =>
+      val json = names.toSet.flatten.map(_.show).mkString("[\"", "\",\"", "\"]")
+      val resp = req.response
+      resp.contentString = json
+      resp.contentType = MediaType.Json
+      resp
+    }.values.toFuture.flatMap(Future.const)
```

Comment:
tioli: i'd be inclined to write this as

`Activity.collect(namers.map(_.getAllNames)).values.toFuture.flatMap(Future.const).map { names =>`

(You don't really want an Activity of responses ever -- you want to get a list of names and then build a response). Not a huge deal.

Also nit: we use `rsp` throughout linkerd.  Prefer `rsp` over `resp` (since this isn't go ;p)